yajsapi/runner/index.ts

```diff
@@ -719,21 +728,27 @@ export class Engine {
           for (let worker_task of [...workers]) {
             worker_task.cancel();
           }
-          // await asyncio.wait(workers, timeout=15, return_when=asyncio.ALL_COMPLETED)
+          await bluebird.Promise.any([
+            bluebird.Promise.all([...workers]),
+            promise_timeout(10)
+          ]);
```

Comment:
I don't know `bluebird.Promise`, but this looks suspicious: shouldn't it be `bluebird.Promise.all(...)`?